src/style.css

```diff
@@ -0,0 +1,125 @@
+.body {
+  margin: 0;
+}
+
+.product {
+  font-family: "Roboto", sans-serif;
```

Comment:
move to `html` selector to prevent code duplicating

Reply:
now you can remove it here

Reply:
not fixed